src/main/java/io/charkchalk/backend/json/JsonConverter.java

```diff
@@ -32,4 +33,9 @@ public static List<FieldNotValidItem> checkPageableSortProperty(Pageable pageabl
 
         return fieldNotValidItems;
     }
+
+    public static String generateSlug(String name) {
+        String randomString = UUID.randomUUID().toString().replace("-","").substring(0,8);
```

Comment:
I would prefer to save full UUID for further possibilities like duplication

Reply:
btw, I mean to save full in database but be able to query by only first 8 digits

Reply:
btw, I mean to save full in database but be able to query by only first n digits

Reply:
I see, but wait for the first version of the specification to be completed before discussing